src/qp/metrics/pit.py

```diff
@@ -0,0 +1,181 @@
+import logging
+import numpy as np
+from scipy import stats
+import qp
+from qp.metrics.metrics import calculate_outlier_rate
+from qp.metrics.array_metrics import quick_anderson_ksamp, quick_cramer_von_mises, quick_kolmogorov_smirnov
+
+DEFAULT_QUANTS = np.linspace(0, 1, 100)
+
+class PIT():
+    """PIT(qp_ens, ztrue, eval_grid=DEFAULT_QUANTS)
+    Probability Integral Transform
+
+    Args:
+        qp_ens Ensemble: A collection of N distribution objects
+        ztrue [float]: An array-like sequence of N float values representing the known true value for each distribution
+        eval_grid ([float], optional): A strictly increasing array-like sequence in the range [0,1]. Defaults to DEFAULT_QUANTS.
+    """
+
+    def __init__(self, qp_ens, ztrue, eval_grid=DEFAULT_QUANTS):
+        """We will create a quantile Ensemble to store the PIT distribution, but also store the
+        full set of PIT samples as ancillary data of the (single PDF) ensemble.
+
+        Args:
+            qp_ens Ensemble: A collection of N distribution objects
+            ztrue [float]: An array-like sequence of N float values representing the known true value for each distribution
```

Comment:
For the purposes of qp, we should probably call this `true_vals` or something like that because it won't always be redshift.